packages/browser/types/test/types.test.js

```diff
@@ -1,16 +1,12 @@
-const { describe, it, expect, afterAll } = global
+const { describe, it, expect } = global
 
 const { spawnSync } = require('child_process')
-const cleanupFiles = []
-
-afterAll(() => {
-  cleanupFiles.forEach(f => spawnSync('rm', ['-f', `${__dirname}/fixtures/${f}`]))
-})
 
 const assertTsProgramCompiles = p => {
-  cleanupFiles.push(`${p}.js`)
   const { stdout, stderr, err } = spawnSync('./node_modules/.bin/tsc', [
     '--strict',
+    '--noEmit',
+    '--skipLibCheck',
```

Comment:
When adding jest to the monorepo a number of @types packages were updated which include new TS functionality such as `unknown`. One way to get around it is to not type check node_modules using `--skipLibCheck`. Alternatively updating to TypeScript > 3 fixes the issues. Let me know which approach you prefer @bengourley. Of the two I think bumping to the latest TypeScript is best but I can see an argument for doing both as there's no need to type check node_modules for these particular tests.

Reply:
I think we should update to TypeScript v3